src/components/DropZone/DropZone.story.jsx

```diff
@@ -0,0 +1,52 @@
+import { select, text, withKnobs, number } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { specs } from 'storybook-addon-specifications';
+import { defaultValues } from '../../constants/defaultValues';
+import { Component, tests } from './DropZone.test';
+import { DropZone } from './DropZone';
+
+const stories = storiesOf('DropZone', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add(
+    'Overview',
+    () => {
+        return (
+            <>
+                <DropZone />
+            </>
+        );
+    },
+    {
+        info: { inline: true },
+    }
+);
+
+stories.add(
+    'Playground',
+    () => {
+        const size = select(
+            `size:`,
+            ['small', `medium`, 'large'],
+            defaultValues.size
+        );
+        const type = select(
+            `type:`,
+            ['primary', 'success', 'warning', 'error'],
+            defaultValues.type
+        );
+        const state = select(`state:`, ['default', 'uploading'], 'default');
+        const progress = number(`progress: `, 0);
```

Comment:
To tu už nepotřebujeme. Raději bych to smazal :) + všechno, co na tuto proměnnou navazuje. Kdyžtak na to mrknem spolu ;)